packages/web-api/src/WebClient.spec.js

```diff
@@ -319,7 +320,21 @@ describe('WebClient', function () {
           const warnClient = new WebClient(token, { logLevel: LogLevel.WARN, logger });
           return warnClient.apiCall(method, args)
             .then(() => {
-              assert.isTrue(logger.warn.callCount === 4);
+              // assume no warning about thread_ts has been sent
```

Comment:
This test is unrelated related to the added features for files.uploadV2 support. It was failing when I added additional warnings re: ☝️. I realized that the test wasn't really specifically validating that the thread_ts warning was being logged, just counting the total number of overall warning loggings - this change should be a small improvement.